backend/shared/contributors.js

```diff
@@ -0,0 +1,5 @@
+const linkedIn = {
+    'sophyphreak': 'https://www.linkedin.com/in/sophyphreak/'
```

Comment:
add the current contributors
```suggestion
    'sophyphreak': 'https://www.linkedin.com/in/sophyphreak/'
    'monarchwadia': 'https://www.linkedin.com/in/monarchwadia/',
    'leanlunio': 'https://www.linkedin.com/in/leanjunio/',
    'shohei-shibata:' 'https://www.linkedin.com/in/shohei-shibata-53867a41/',
    'carloshdelreal': 'https://www.linkedin.com/in/carloshdelreal/',

```